### PR TITLE
[DOCS-6373] ASM Signals Explorer doc

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do? What is the motivation?
This PR creates a new ASM Signals explorer doc, as we are archiving the [existing](https://docs.datadoghq.com/security/explorer) high level doc under Security Overview.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/application_security/threats/security_signals.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: Investigate Security Signals
+kind: documentation
+further_reading:
```
Comment (datamarmot) [inline content/en/security/application_security/threats/security_signals.md:4]: Lines 4 : 10 links to the CSM product which is distinct from ASM.  @aliciascott 
Need to point to ASM.

File: content/en/security/application_security/threats/security_signals.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    tag: "Documentation"
+    text: "Learn how to manage CSM Threats detection rules"
+  - link: "/security/notifications/"
+    tag: "Documentation"
+    text: "Learn more about security notifications"
+---
+
+## Overview
+
+ASM security signals are created when Datadog detects a threat based on a security rule. View, search, filter, and investigate security signals in the [Signals Explorer][2], or configure [Notification Rules][8] to send signals to third-party tools.
```
Comment (datamarmot) [inline content/en/security/application_security/threats/security_signals.md:18]: > ASM security signals are created when Datadog detects a threat based on a security rule
Change security rule to detection rule. Detection Rules is the term used in the UI. [example](https://app.datadoghq.com/security/appsec/detection-rules?query=type%3Aapplication_security&deprecated=hide&groupBy=none&sort=rule_name):

File: content/en/security/application_security/threats/security_signals.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+4. On the workflow modal, select the workflow you want to run. Depending on the workflow, you may be required to enter additional input parameters.
+5. Click **Run**.
+
+## Review and remediate
+
+1. On the [Signals Explorer][2], select a security signal.
+2. On the signal side panel, click each of the review tabs such as **Attack Flow**, **Activity Summary**, and **Rule Details**.
+3. Review the **Suggested Next Steps**, and take action:
+    -  Click to **Block attacking users and IPs**
+    -  Click to **Block with Edge WAF**.
```
Comment (datamarmot) [inline content/en/security/application_security/threats/security_signals.md:75]: Below line 75 add
- Click to **Automated Attacker Blocking**

^ this is new

Comment (aliciascott) [reply content/en/security/application_security/threats/security_signals.md:75]: thanks, I will update the screen shot as well!

File: content/en/security/application_security/threats/security_signals.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+1. On the [Signals Explorer][4], select a security signal.
+2. Scroll down to the **What is Workflow Automation** section. 
+3. Click **Run Workflow**.
+4. On the workflow modal, select the workflow you want to run. Depending on the workflow, you may be required to enter additional input parameters.
+5. Click **Run**.
+
+## Review and remediate
+
+1. On the [Signals Explorer][2], select a security signal.
+2. On the signal side panel, click each of the review tabs such as **Attack Flow**, **Activity Summary**, and **Rule Details**.
```
Comment (maycmlee) [inline content/en/security/application_security/threats/security_signals.md:72]:
```suggestion
2. On the signal side panel, click each of the tabs, such as **Attack Flow**, **Activity Summary**, and **Rule Details**, to review the information.
```

**Review by maycmlee — APPROVED** [review]
Left a couple of small suggestions, otherwise looks good to me! I'm approving, but feel free to let me know if you want a re-review.

File: config/_default/menus/menus.en.yaml [hunk 5]
```diff
@@ -3727,11 +3727,16 @@ main:
...
+    weight: 507
   - name: Trace Qualification
     url: security/application_security/threats/trace_qualification/
     parent: appsec_threats
     identifier: threats_trace_qualification
-    weight: 507
+    weight: 508
   - name: Threat Intelligence
     url: security/application_security/threats/threat-intelligence/
     parent: appsec_threats
```
[the remark this answers is not in the document]
Comment (aliciascott) [reply config/_default/menus/menus.en.yaml:3742]: oh I may have missed this, thanks let me take a look !

**Pull request merged**